docs/sphinx/reference-frontend-api.rst

```diff
@@ -595,3 +599,27 @@ Functions
 .. function:: void obs_frontend_reset_video(void)
 
    Reloads the UI canvas and resets libobs video with latest data from profile.
+
+---------------------------------------
+
+.. function:: void obs_frontend_set_tbar_position(int position)
+
+   Set the value of the transition bar.
+
+   :param position: The position to set the T-bar to, with a value in 0-1023.
+   :type position: int
+
+---------------------------------------
+
+.. function:: void obs_frontend_release_tbar(void);
+
+   Emulate a mouse button release on the transition bar and determine transition status.
+
+---------------------------------------
+
+.. function:: int obs_frontend_get_tbar_position(void)
+
+   Get the value of the transition bar.
+
+   :return: The value of the position of the T-bar to, with a value in 0-1023.
+   :rtype: int
```

Comment:
Nitpick:  These are in the following order in `UI/api-interface.cpp`, `UI/obs-frontend-api/obs-frontend-api.cpp`, and `UI/obs-frontend-api/obs-frontend-api.h`:
```
obs_frontend_release_tbar
obs_frontend_set_tbar_position
obs_frontend_get_tbar_position
```

That should hopefully be my final nitpick.